packages/mdl-ripple/index.js

```diff
@@ -39,67 +21,123 @@ export default class MDLRipple {
     background.classList.add(Class.BACKGROUND);
     root.appendChild(background);
 
-    this.elements_ = {
-      [Identifier.ROOT]: root,
-      [Identifier.BACKGROUND]: background,
-      [Identifier.SURFACE]: surface,
-    };
-
     for (let i = 0; i < MAX_RIPPLES; i++) {
       let foreground = document.createElement('div');
       foreground.classList.add(Class.FOREGROUND);
       let foregroundCircle = document.createElement('div');
       foregroundCircle.classList.add(Class.FOREGROUND_CIRCLE);
       foreground.appendChild(foregroundCircle);
       root.appendChild(foreground);
+    }
+    return root;
+  }
 
-      this.elements_[this.get(Identifier.FOREGROUND, i)] = foreground;
-      this.elements_[this.get(Identifier.FOREGROUND_CIRCLE, i)] = foregroundCircle;
+
+  /**
+   * @param {!Element} surface
+   * @param {!Element} root
+   * @return {!MDLRipple}
+   */
+  static attachTo(surface, root) {
+    let elements = {
+      [Identifier.SURFACE]: surface,
+      [Identifier.ROOT]: root,
+      [Identifier.BACKGROUND]: root.getElementsByClassName(Class.BACKGROUND)[0]
```

Comment:
This one I'd like to keep, as I prefer using the specific DOM api if it's available.